### PR TITLE
fix multiple return error case; fix expire time unit issue

### PR DESCRIPTION
**Pull request by plateaukao**

### CODE REVIEW

File: android/src/main/kotlin/com/linecorp/flutter_line_sdk/model/AccessToken.kt [hunk 1]
```diff
@@ -21,7 +21,7 @@ data class AccessToken(
             val accessToken = loginResult.lineCredential?.accessToken ?: return null
             return AccessToken(
                 accessToken.tokenString,
-                accessToken.expiresInMillis,
+                (accessToken.estimatedExpirationTimeMillis - System.currentTimeMillis()) / 1000,
                 lineIdTokenString
```
Comment (onevcat) [inline android/src/main/kotlin/com/linecorp/flutter_line_sdk/model/AccessToken.kt]: To algin the [documentation](https://pub.dev/documentation/flutter_line_sdk/latest/flutter_line_sdk/AccessToken/expiresIn.html) and iOS implementation, please return the `expires_in` which we received from server when issuing the token. (The number is fixed and counted from when the server issued the token).

Comment (onevcat) [reply android/src/main/kotlin/com/linecorp/flutter_line_sdk/model/AccessToken.kt]: Here is [another explanation](https://pub.dev/documentation/flutter_line_sdk/latest/flutter_line_sdk/StoredAccessToken/expiresIn.html) on how to use these `expires_in` values. Basically, we just provide what received from the server as a simple relaying, and leave the framework users to determine how they would want to use or store them.

Comment (plateaukao) [reply android/src/main/kotlin/com/linecorp/flutter_line_sdk/model/AccessToken.kt]: Okay!

File: android/src/main/kotlin/com/linecorp/flutter_line_sdk/LineSdkWrapper.kt [hunk 2]
```diff
@@ -116,11 +113,9 @@ class LineSdkWrapper(
...
 
         if (resultCode != Activity.RESULT_OK || intent == null) {
-            loginResult?.error(
-                resultCode.toString(),
-                "login failed",
-                null
-            )
+            loginResult?.error(resultCode.toString(), "login failed", null)
+            loginResult = null
+            return true
```
Comment (tsairene) [inline android/src/main/kotlin/com/linecorp/flutter_line_sdk/LineSdkWrapper.kt]: 👍

**Pull request merged**